apps/explorer/src/components/orders/StatusLabel/styled.ts

```
@@ -0,0 +1,155 @@
+      break
+    case BridgeStatus.Completed.toLowerCase():
+    case BridgeStatus.RefundComplete.toLowerCase():
+      text = Color.explorer_green1
+      background = Color.explorer_greenOpacity
+      break
+    case BridgeStatus.Failed.toLowerCase():
+    case BridgeStatus.Refunding.toLowerCase():
+      text = Color.explorer_orange1
+      background = Color.explorer_orangeOpacity
+      break
+    case BridgeStatus.Unknown.toLowerCase():
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)